content/administration/upgrade/odoo_sh.rst

```diff
@@ -1,46 +1,72 @@
 
-=======================
-Odoo.sh version upgrade
-=======================
+=======
+Odoo.sh
+=======
 
-Download and Upload your database
-=================================
+Test upgrade request
+====================
 
-Download a dump of your database (from the :ref:`Builds view <odoosh-gettingstarted-builds-download-dump>`), choose the
-exact copy and without filestore options. Upload the .sql.gz dump on https://upgrade.odoo.com/upload and
-select the Testing Purpose. If you have custom code, you can choose to have it upgraded by us, or do it yourself. Once
-it's processed, you'll get a dump of the database in return.
+Download a dump of your database (from the :ref:`Builds view <odoosh-gettingstarted-builds-download-dump>`),
+choose the **exact copy and without filestore** options. Upload the .sql.gz dump on https://upgrade.odoo.com/upload and
+select the testing purpose. Once it's processed, you'll get a dump of the database in return.
 
-.. warning::
-   Do *not* upload *backups* of your production database (found in the Backups tab of the production branch) as these
-   are incompatible with the Upgrade platform - they contain your complete sources, etc. that are not needed for the
-   upgrade. Make sure to download a **Dump** instead - either through the Backups tab using the *Download Dump* button
-   or through the Builds page by using the *Download Dump* entry of the contextual menu of your latest production build.
+.. important::
+   To have the most up to date test version of your database, create a back-up of your
+   production database just before creating your request. Upload it in your staging branch
+   selecting **Exact dump** and **Without filestore** and then click start.
```

Comment:
- 100th character line break
- up-to-date --> hyphens
- backup
- grammar tenses on the last sentence


```suggestion
.. important::
   To have the most up-to-date test version of your database, create a backup of your production
   database just before creating your request. Upload it in your staging branch, select **Exact
   dump** and **Without filestore**, and then click on *start*.
```